components/inputs/test/input-number.test.js

```diff
@@ -455,6 +455,7 @@ describe('d2l-input-number', () => {
 			const elem = await fixture(minMaxFixture);
 			setTimeout(() => setInnerInputValue(elem, '0'));
 			await oneEvent(elem, 'change');
```

Comment:
Could we `await` an `invalid-change` event here instead of the `change` and `updateComplete`?